loki/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "@perses-dev/loki-plugin",
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "rsbuild dev",
+   "build": "npm run build-mf && concurrently \"npm:build:*\"",
+    "build-mf": "rsbuild build",
+    "build:cjs": "swc ./src -d dist/lib/cjs --strip-leading-paths --config-file ../.cjs.swcrc",
+    "build:esm": "swc ./src -d dist/lib --strip-leading-paths --config-file ../.swcrc",
+    "build:types": "tsc --project tsconfig.build.json",
+    "test": "cross-env LC_ALL=C TZ=UTC jest",
+    "type-check": "tsc --noEmit"
+  },
+  "main": "lib/cjs/index.js",
+  "module": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "devDependencies": {
```

Comment:
as you are adding this new package into a npm workspace, you don't need these dev dependencies as they are already installed.